biojava-survival/src/main/java/org/biojava/nbio/survival/cox/matrix/StdArrayIO.java

```diff
@@ -91,7 +91,7 @@ public static void print(double[] a) {
 		LOGGER.info(String.valueOf(N));
 		for (int i = 0; i < N; i++) {
 		 //   System.out.printf("%9.5f ", a[i]);
-			System.out.print(a[i] + " ");
+			LOGGER.info(a[i] + " ");
```

Comment:
This breaks the formatting with new lines done on stdout. This should be done with a StringBuffer that is then printed to a single log statement